cegs_portal/search/tsv_templates/v1/non_targeting_reos.py

```diff
@@ -1,11 +1,42 @@
+from cegs_portal.search.templatetags.custom_helpers import format_significance
+
+
 def non_targeting_regulatory_effects(data):
     reo_page, feature = data
     tsv_data = []
-    tsv_data.append(['Chromosome Location', 'Effect Size', 'Direction', 'Significance', 'Distance', 'Feature Type', 'Experiment'])
+    tsv_data.append(
+        [
+            "chrom",
+            "chromStart",
+            "chromEnd",
+            "name",
+            "strand",
+            "score",
+            "Effect Size",
+            "Direction",
+            "Significance",
+            "Distance",
+            "Feature Type",
+            "Experiment",
+        ]
+    )
 
     for reo in reo_page:
         first_source = reo.sources.all()[0]
-        row = [first_source.chrom_name, first_source.location.lower, first_source.location.upper, reo.effect_size, reo.direction, reo.significance, first_source.closest_gene_distance, first_source.get_feature_type_display(), reo.experiment.name]
+        row = [
+            first_source.chrom_name,
+            first_source.location.lower,
+            first_source.location.upper,
+            feature.name,
+            feature.strand,
+            ".",
+            reo.effect_size,
+            reo.direction,
+            format_significance(reo.significance),
+            first_source.closest_gene_distance,
+            first_source.get_feature_type_display(),
+            reo.experiment.name,
+        ]
```

Comment:
I think this should be nested loops. 

```python
for reo in reo_page:
    for source in reo.sources.all():
        row = [
            source.chrom_name,
            source.location.lower,
            source.location.upper,
            feature.name,
            feature.strand,
            ".",
            format_float(reo.effect_size),
            reo.direction,
            format_float(reo.significance),
            source.closest_gene_distance,
            source.get_feature_type_display(),
            reo.experiment.name,
            reo.accession_id,
        ]
```

With an extra column of the REO accession id that will allow users to tie multiple "sources" to the same REO.

Reply:
Completed and will update other branch